admin/model_search_index_update_request.go

```diff
@@ -0,0 +1,68 @@
+// Code based on the AtlasAPI V2 OpenAPI file
+
+package admin
+
+import (
+	"encoding/json"
+)
+
+// SearchIndexUpdateRequest struct for SearchIndexUpdateRequest
+type SearchIndexUpdateRequest struct {
+	// Fields to index for vector search.
+	Definition map[string]interface{} `json:"definition"`
+}
```

Comment:
`Definition` property in this case is untyped, but should point to a struct that combines ApiAtlasTextSearchIndexDefinitionView and ApiAtlasVectorSearchIndexDefinitionView (https://github.com/10gen/mms/blob/master/server/src/main/com/xgen/cloud/search/api/_public/view/api/ApiAtlasSearchIndexUpdateRequestView.java#L13)

Reply:
This have been fixed upstream.